File: src/kyu_7/sum_of_cubes/cpp/sum_of_cubes_test.cpp
Describe(Test_Cases) {
  It(Fixed_Tests) {
    Assert::That(sum_cubes(1), Equals(1));
    Assert::That(sum_cubes(2), Equals(9));
    Assert::That(sum_cubes(3), Equals(36));
    Assert::That(sum_cubes(4), Equals(100));
    Assert::That(sum_cubes(10), Equals(3025));
    Assert::That(sum_cubes(123), Equals(58155876));
  }
};

File: src/kyu_7/sum_of_cubes/python/sum_of_cubes_test.py
import codewars_test as test
from solution import sum_cubes

@test.describe("Sum of Cubes")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(sum_cubes(1), 1)
        test.assert_equals(sum_cubes(2), 9)
        test.assert_equals(sum_cubes(3), 36)
        test.assert_equals(sum_cubes(4), 100)
        test.assert_equals(sum_cubes(10), 3025)
        test.assert_equals(sum_cubes(123), 58155876)
        